qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,108 @@
+Feature: 1.1.2 Send_CV.feature
+
+Background: 
+	Given user accessed Quantori Career Portal from his browser
+	And user scrolled down to CV sending form
+
+Scenario: 1.1.2_01 User can successfully send CV
+	Given user inputted "Ivan Ivanov" in field "Name"
+	And user inputted "ivan.ivanov@gmail.com" in field "Email"
+	And user attached CV with file format being PDF/DOC/DOCX/RTF with size less than 10 Mb
+	And user checked Checkbox
+	When user clicks Send button
+	Then message box "Thanks for being awesome! We have received your message and will come back to you as soon as we process it." should appear
+	And form should be empty
+
+Scenario: 1.1.2_02 User can't send empty form
```

Comment:
I guess, that scripts 1.1.2_02, 1.1.2_05, 1.1.2_06, 1.1.2_10 can be combined into one script **and** add an example where the user does not attach the CB. For example:
Scenario Outline: 1.1_02 If important fields are empty, no sending occurs, errors are highlighted
![Screenshot from 2021-12-02 02-34-10](https://user-images.githubusercontent.com/95170810/144331310-dcd96001-b5ad-4a50-acd5-fdd998040df1.png)